src/tools/pub/dependencies.md

```diff
@@ -429,33 +429,50 @@ depend on that package.
 These stale versions can have a negative impact on
 the stability, performance, and quality of apps.
 
-We recommend the following best practices for package dependencies:
-
-Use [caret syntax](#caret-syntax)
-: Specifying dependencies with version ranges is such as `^1.6.3`
-  is a good practice because it allows the pub tool to
-  select newer versions of the package when they become available.
-  Also, it places an upper limit on the allowed version,
-  based on an assumption that packages use [semantic versions][],
-  where any version of path versioned `1.x` is compatible,
-  but where a new version `2.x` would be a major upgrade
-  that isn't semantically compatible with `1.x` versions. 
-
-Depend on the latest stable package versions
-: Use [`pub upgrade`][] to update to the latest package versions
-  that your pubspec allows.
-  To identify dependencies in your app or package that
-  aren't on the latest stable versions,
-  use [`pub outdated`][].
-
-Test whenever you update package dependencies
-: If you run [`pub upgrade`][] without updating your pubspec,
-  the API should stay the same
-  and your code should run as before—but test to make sure.
-  If you modify the pubspec and update to a new major version,
-  then you might encounter breaking changes,
-  so you need to test even more thoroughly.
-
+We recommend the following best practices for package dependencies.
+
+### Use [caret syntax](#caret-syntax)
+
+Specifying dependencies with version ranges is such as `^1.6.3`
+is a good practice because it allows the pub tool to
+select newer versions of the package when they become available.
+Also, it places an upper limit on the allowed version,
+based on an assumption that packages use [semantic versions][],
+where any version of path versioned `1.x` is compatible,
+but where a new version `2.x` would be a major upgrade
+that isn't semantically compatible with `1.x` versions. 
+
+### Depend on the latest stable package versions
+
+Use [`dart pub upgrade`][] to update to the latest package versions
+that your pubspec allows.
+To identify dependencies in your app or package that
+aren't on the latest stable versions,
+use [`dart pub outdated`][].
+
+### Test whenever you update package dependencies
+
+If you run [`dart pub upgrade`][] without updating your pubspec,
+the API should stay the same
+and your code should run as before—but test to make sure.
```

Comment:
**issue** The way this is written sounds like you are not encouraging testing. If you are, start with that, then say that the API and code should be unaffected anyway.